exercises/collatz-conjecture/collatz_conjecture_test.f90

```diff
@@ -0,0 +1,35 @@
+
+! This test was created from E:\nosave\code\gitrepos\pclausen\problem-specifications\exercises\collatz-conjecture\canonical-data.json
```

Comment:
You might want to strip the first part of the path from this comment as it's specific to your computer.